### PR TITLE
Edit for style, lint, and readability

### PR DESCRIPTION
**Pull request by philoserf**

### CODE REVIEW

File: bin/skopos [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-source $SKOPOS
-skopos $@
+source "$SKOPOS"
```
Comment (philoserf) [inline bin/skopos:3]: what if these are not defined?

File: install [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```
Comment (philoserf) [inline install:1]: I'd prefer that skopos stood alone. That it didn't depend upon shell config and assume things about a users home directory setup. I have set a subconscious time go routine in my mind to mull over a concrete recommendation on getting there.

File: README.md [hunk 3]
```diff
@@ -1,21 +1,20 @@
...
-<name>] [remove <name>] [help]`
+`skopos [init <name>] [list] [switch <name>] [create <name>] [remove <name>] [help]`
 
 ## Description
 
-c|create : Creates a new skopos env and switches to it. i|init :
-Initialize new skopos env. l|ls|list : List all kraken environments
-available. s|sw|switch : Switch to kraken environment. r|rm|remove :
-Explains how to remove an environment. h|help : This message.
+- c|create: Creates a new skopos env and switches to it
```
Comment (philoserf) [inline README.md]: It is not clear in the README is `create` is a combination of `init` and `switch`.

Comment (notjames) [reply README.md]: good call. will fix.

**Review by notjames — APPROVED** [review]

File: bin/skopos (hunk 1 above)
Comment (notjames) [reply bin/skopos:3]: I'm not sure I understand what you mean by "these" in this context? This is a good change, though. Ah, I think I just got it...where is the sanity check, correct? I'll handle.

File: install (hunk 2 above)
Comment (notjames) [reply install:1]: had OOB conversation. Good chat. I'll be thinking through other possibles as well as looking at other projects' methodologies for ideas.

**Review by notjames — APPROVED** [review]
lgtm

**Pull request merged**